### ISSUE TITLE
[classnames] ClassDictionary should have value `any`

### ISSUE THREAD

**Issue by jeffijoe** [DefinitelyTyped/DefinitelyTyped#11317]
I want to talk about `classnames/classnames.d.ts`.
  - The authors of that type definition are cc/ @vvakame @JKillian

```ts
interface ClassDictionary {
	[id: string]: boolean;
}
```

should be

```ts
interface ClassDictionary {
	[id: string]: any;
}
```

or at least

```ts
interface ClassDictionary {
	[id: string]: boolean | undefined;
}
```

This makes it easier when defining your props like so:

```ts
interface IMyProps {
  small?: boolean;
  medium?: boolean;
}
```

What do you think?

I would submit the patch myself but I am not set up to run the tests.

**Comment by vvakame** [comment]
please send a pull request. I'll review it.

**Comment by jeffijoe** [comment]
It appears to have been implemented - issue #11584
